api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -150,7 +152,55 @@ public String getGlobalProperty(String propertyName) throws APIException {
 			return null;
 		}
 		
-		return dao.getGlobalProperty(propertyName);
+		String globalProperty = dao.getGlobalProperty(propertyName);
```

Comment:
Why do we call both `dao.getGlobalProperty()` and `dao.getGlobalPropertyObject()`? Can't we just call one of the two?

Reply:
@dkayiwa sure let me check if it's possible